src/seedu/addressbook/parser/Parser.java

```diff
@@ -75,33 +76,36 @@ public Command parseCommand(String userInput) {
 
         switch (commandWord) {
 
-        case AddCommand.COMMAND_WORD:
-            return prepareAdd(arguments);
+            case AddCommand.COMMAND_WORD:
```

Comment:
Tons of readjustment here that is not very relevant to the PR you've made. Like before, try not to have too many repositioning unless really necessary.